frontend/src/components/VPopover/meta/VPopover.stories.mdx

```diff
@@ -89,7 +85,35 @@ const TwoPopoverStory = (args, { argTypes }) => ({
     </div>
   `,
   components: { VPopover, VButton },
+  setup() {
+    return { args }
+  },
 })
 
-export const TwoPopovers = TwoPopoverStory.bind({})
-TwoPopovers.args = {}
+<Description of={VPopover} />
+
+<ArgsTable of={VPopover} />
+
+## Default VPopover
+
+<Canvas>
+  <Story name="Default" argTypes={popoverArgTypes}>
```

Comment:
If the arg types are repeated, they can be moved to the `<Meta>` tag at the top of the file.

```suggestion
  <Story name="Default" >
```